applications/nrf5340_audio/prj_fota.conf

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
This is the duplicated config that must remain like this for now?

Reply:
This isn't just for now - the file has to stay duplicated. The entire `FILE_SUFFIX` feature is based on the `prj.conf` being different for each build type.